rasa/core/nlg/callback.py

```diff
@@ -27,42 +27,19 @@ def nlg_response_format_spec() -> Dict[Text, Any]:
     }
 
 
-def nlg_request_format_spec() -> Dict[Text, Any]:
-    """Expected request schema for requests sent to an NLG endpoint."""
-
-    return {
-        "type": "object",
-        "properties": {
-            "template": {"type": "string"},
-            "arguments": {"type": "object"},
-            "tracker": {
-                "type": "object",
-                "properties": {
-                    "sender_id": {"type": "string"},
-                    "slots": {"type": "object"},
-                    "latest_message": {"type": "object"},
-                    "latest_event_time": {"type": "number"},
-                    "paused": {"type": "boolean"},
-                    "events": {"type": "array"},
-                },
-            },
-            "channel": {"type": "object", "properties": {"name": {"type": "string"}}},
-        },
-    }
-
-
 def nlg_request_format(
-    template_name: Text,
+    utter_action: Text,
     tracker: DialogueStateTracker,
     output_channel: Text,
     **kwargs: Any,
 ) -> Dict[Text, Any]:
     """Create the json body for the NLG json body for the request."""
-
     tracker_state = tracker.current_state(EventVerbosity.ALL)
 
+    # Template is deprecated, please use response instead.
```

Comment:
I suggest to add a TODO to remove this in 3.0.0. Otherwise we'll have to drag this around forever + add this to the changelog.